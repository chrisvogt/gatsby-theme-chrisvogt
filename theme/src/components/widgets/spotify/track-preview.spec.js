import React from 'react'
import renderer from 'react-test-renderer'
import TrackPreview from './track-preview'

describe('TrackPreview', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <TrackPreview
          link='https://www.fake-book-website.com/book-example'
          name='Fake Song'
          thumbnailURL='https://placehold.it/400/400'
        />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
