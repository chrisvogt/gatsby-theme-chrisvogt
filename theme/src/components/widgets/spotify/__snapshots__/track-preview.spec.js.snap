// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`TrackPreview matches the snapshot 1`] = `
<a
  className="css-zezy6m"
  href="https://www.fake-book-website.com/book-example"
  title="Fake Song"
>
  <img
    alt="album cover"
    className="css-o32fug"
    crossOrigin="anonymous"
    loading="lazy"
    src="https://placehold.it/400/400"
  />
</a>
`;
